doc/strategies/LimitOrder.md

```diff
@@ -0,0 +1,28 @@
+
+Besides the traditional taker scenario, an order can also be satisfied by liquidity from a supported AMM protocol. It relies on someone(EOA) trying to match the price of AMM protocols and the opened orders. In order to incentivize relayers to participate, the difference of the price is considered as the profit of a relayer.
+
+To avoid multiple filling transactions for the same order colliding because insufficient amount is left, the coordinator design is introduced. The idea is that every fill should be signed by a coordinator so the available amount can be calculated as early as possible.
+
+## Order Format
+
+| Field            |  Type   | Description                                                                     |
+| ---------------- | :-----: | ------------------------------------------------------------------------------- |
+| makerToken       | address | The address of the maker token.                                                 |
+| takerToken       | address | The address of the taker token.                                                 |
+| makerTokenAmount | uint256 | The amount of the maker token.                                                  |
+| takerTokenAmount | uint256 | The amount of the taker token.                                                  |
+| maker            | address | The address of the maker.                                                       |
+| taker            | address | The address of the taker. Set to zero address if no specific taker is assigned. |
+| salt             | uint256 | A random number included in an order to avoid replay attack.                    |
+| expiry           | uint64  | The timestamp of the expiry.                                                    |
+
+## Order Cancelling
+
+If a maker wants to cancel an opened order, a special signature is required. More specifically, the maker should modify the original order with `takerTokenAmount` set to zero and sign it. By submitting cancelling signature to the contract, the original order is marked as canceled which can not by filled anymore.
+
+## Fee
+
+There are totally three different fee factors in `LimitOrder` contract. For `fillByTrader` scenario, `makerFeeFactor` and `takerFeeFactor` can be applied to calculate two kinds of fees. For `fillByProtocol` case, the `takerFeeFactor` is applied to the taker asset as well. However, there's another `profitFeeFactor` for the profit of the relayer. All of the fees will be transferred to `feeCollector` during the settlement.
```

Comment:
```suggestion
There are totally three different fee factors in `LimitOrder` contract. For `fillByTrader` scenario, `makerFeeFactor` and `takerFeeFactor` will be applied to calculate two kinds of fees. For `fillByProtocol` case, only the `takerFeeFactor` is applied to the taker asset. However, there's another `profitFeeFactor` to calculate the profit for the relayer. All of the fees will be transferred to `feeCollector` during the settlement.
```

Reply:
I think this can be left for formal grammar correction.